helm/harvester/charts/harvester/panda_harvester_configmap.json

```diff
@@ -32,9 +32,16 @@
     "type": "apache"
   },
   "qconf": {
-    "configFile": "/etc/harvester/queueconfig/panda_queueconfig.json",
+    "configFile": "/opt/harvester/etc/panda/panda_queueconfig.json",
```

Comment:
You mount the json under /opt/harvester/etc/queue_config at https://github.com/PanDAWMS/panda-k8s/blob/aecf48c5b0070d1757ea34b7a4731e84a1d2bce0/helm/harvester/charts/harvester/templates/deployment.yaml#L99

Why the path is /opt/harvester/etc/panda?